go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/pvotal-tech/terraform-provider-k3d
 
 go 1.18
 
+replace github.com/k3d-io/k3d/v5 => github.com/moio/k3d/v5 v5.0.0-20230328124135-da29b7058f1e
```

Comment:
We ll have to make sure to return to k3d-io for security dependences once moio's fix upstream is approved

Reply:
See #78